app.py

```diff
@@ -194,6 +199,36 @@ def get_snap_id(snap_name):
     return details_response.json()['snap_id']
 
 
+def calculate_color(thisCountry, maxCountry, maxColor, minColor):
+    countryFactor = float(thisCountry)/maxCountry
+    colorRange = maxColor - minColor
+    return int(colorRange*countryFactor+minColor)
+
+
+def calculate_colors(countries, max_users):
+    for country_code in countries:
+        countries[country_code]['color_rgb'] = [
+            calculate_color(
+                countries[country_code]['percentage_of_users'],
+                max_users,
+                8,
+                229
```

Comment:
What are these magic numbers, are these colour definitions?

Reply:
I believe so - @tbille?

In this PR I just moved the code so it could be used twice.

Reply:
yes here we define the max and min colors
Same here I don't like that we do this in the backend. We need to have a discussion about what should be send to the frontend...